### PR TITLE
The Importance of Being Design First

### PR DESCRIPTION
**Pull request by poteto**
My first blog post! :)

I had a late night idea (plus I couldn't sleep, damn you jet lag) on what I think differentiates DY from the competition. I don't think its solely because we're experts in growing technologies such as Ember.js or Elixir, but because we believe in design first. 

This post is basically a summary of my experiences as a designer, developer and startup founder + my attempt at selling DY :p

It's still very much a WIP, so I would appreciate help with:
- General editing
- Polish and making things flow better
- I'm open to reorganizing things and better titles
- I need to add references and links, maybe pictures
- Any domain knowledge on design/engineering you can share

To be rebased

### CODE REVIEW

File: source/posts/2015-04-04-the-user-experience-architect.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+comments: true
+published: true
+summary: "Lauren's musings on the art & science of designing & building beautiful user experiences."
+---
+
+## Defining a new profession for the future of the modern web
+The modern architect has her roots in ancient and medieval history. They
+originated as artisans – master craftsmen such as stone masons and carpenters
+that had risen to the level of master builder. Until recently, there was no
+clear separation between the role of the architect and the engineer – they were
```
Comment (bcardarella) [inline source/posts/2015-04-04-the-user-experience-architect.md]: > there was no clear separation between the role of the architect and the engineer

I agree with this statement WRT web development, I don't think it is as defendable when it is in the broader sense that the paragraph implies. In modern construction there is a clear separation between the architect and the engineer. I like the direction but you may want to refine this some

File: source/posts/2015-04-04-the-user-experience-architect.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+speak.
+
+When you intimately know how a system works, and understand how its constituent
+pieces fit together, you can identify problems early and gain insights into
+matters you might not have considered if you only had a shallow understanding
+of the system. To put it in a familar analogy, a software engineer cannot
+possibly design an elegant and efficient low level programming language without
+a solid understanding of how byte code is compiled into machine code.
+
+## The craftsmens' renaissance
```
Comment (bcardarella) [inline source/posts/2015-04-04-the-user-experience-architect.md]: craftmens' => craftmen's

**Comment by bcardarella** [comment]
:+1: I dig it, the tone is great

**Comment by poteto** [comment]
Closed and reopened a new PR #326

**Pull request closed without merging**